wgpu-hal/src/gles/adapter.rs

```diff
@@ -628,6 +628,11 @@ impl super::Adapter {
             // that's the only way to get gl_InstanceID to work correctly.
             features.set(wgt::Features::INDIRECT_FIRST_INSTANCE, supported);
         }
+        // Supported by GL 3.3+, Not supported by GLES 3.0+
+        features.set(
+            wgt::Features::TEXTURE_FORMAT_16BIT_NORM,
+            supported((0, 0), (3, 3)),
```

Comment:
by putting (0, 0) first you're always enabling this feature for OpenGL ES, instead of disabling it always

I think what you actually want is
```suggestion
      es_ver.is_none() && full_ver >= (3, 3))
```

Reply:
Fixed, required using `Some((3, 3))`.